pyproject.toml

```diff
@@ -284,6 +284,9 @@ dl = [
   'transformers[torch]<4.41.0; python_version < "3.12"',
   'pykan; python_version > "3.9.7"',
   'pytorch-forecasting>=1.0.0; python_version < "3.11"',
+  'lag-llama@git+https://github.com/time-series-foundation-models/lag-llama.git',
+  'gluonts',
```

Comment:
you can remove gluonts and huggingface_hub, since both should also be dependencies of lag llama